mmdet/core/evaluation/text_evaluation.py

```diff
@@ -225,7 +199,7 @@ def parse_gt_objects(gt_annotation, use_transcription):
     return gt_polygons_list, gt_dont_care_polygon_nums, gt_transcriptions
 
 
-def parse_pr_objects(pr_annotation, conf_thr, use_transcription):
+def parse_pr_objects(pr_annotation, conf_thr, use_transcription, word_spotting):
```

Comment:
`word_spotting` is not used inside the function. Why is it here?

Reply:
fixed